### PR TITLE
`MetaBackend`

### PR DESCRIPTION
**Pull request by BrunoLiegiBastonLiegi**
Updates `qiboml` for compatibility with the new `MetaBackend` mechanism introduced in `qibo`

### CODE REVIEW

**Comment by BrunoLiegiBastonLiegi** [comment]
@alecandido @MatteoRobbiati this is ready, it is failing due to the usual `post load cached venv`

**Review by alecandido — APPROVED** [review]

File: src/qiboml/backends/__init__.py [hunk 1]
```diff
@@ -1 +1,49 @@
...
+QIBOML_BACKENDS = ["tensorflow", "pytorch", "jax"]
+QibomlBackend = Union[TensorflowBackend, PyTorchBackend, JaxBackend]
+
+
+class MetaBackend:
+    """Meta-backend class which takes care of loading the qiboml backends."""
+
+    @staticmethod
+    def load(backend: str, **kwargs) -> QibomlBackend:
+        """Loads the qiboml backend.
```
Comment (MatteoRobbiati) [inline src/qiboml/backends/__init__.py:18]:
```suggestion
        """Load the qiboml backend.
```

File: src/qiboml/backends/__init__.py [hunk 2]
```diff
@@ -1 +1,49 @@
...
+        elif backend == "jax":
+            return JaxBackend()
+        else:
+            raise_error(
+                ValueError,
+                f"Backend {backend} is not available. The qiboml backends are {QIBOML_BACKENDS}.",
+            )
+
+    def list_available(self) -> dict:
+        """Lists all the available native qibo backends."""
```
Comment (MatteoRobbiati) [inline src/qiboml/backends/__init__.py:40]:
```suggestion
        """List all the available native qibo backends."""
```

**Review by MatteoRobbiati — APPROVED** [review]
Thanks!

**Pull request merged**